examples/message-queue-integrations/solace/quick_start/README.md

```diff
@@ -0,0 +1,59 @@
+# Description
+This example demonstrates how to run a workflow and interact with it. Two workflows have already been developed in the [deploy_workflow script](./deploy_core.py). The workflow responds to each query with a constant message. You can customize the deploy_workflow to add more features and functionalities to the agent.
+
+## Installation
+### Solace PubSub+ Event Broker
+Install the [Solace PubSub+ event broker](https://docs.solace.com/Get-Started/Getting-Started-Try-Broker.htm).
+
+#### Set Environment Variables
+```bash
+export SOLACE_HOST='tcp://localhost:55554'
+export SOLACE_HOST_SECURED='tcps://localhost:55443'
+export SOLACE_HOST_COMPRESSED='tcp://localhost:55003'
+export SOLACE_VPN_NAME='default'
+export SOLACE_USERNAME='default'
+export SOLACE_PASSWORD='default'
+export SOLACE_IS_QUEUE_TEMPORARY='false'
+```
+
+#### Install Solace PubSub+ Python Package
+```bash
+pip install solace-pubsubplus
+```
+
+#### Verification
+Open the PubSub+ interface in your browser. If you’re running PubSub+ locally, go to http://localhost:8080. Use ‘admin’ for both the username and password. Once the dashboard loads, click on ‘Try Me’ from the left-side menu. Then, subscribe to the 'hello_workflow', 'ping_workflow' and 'control_plane' topics to monitor events.
+
+## Run
+Deploy the core on a terminal.
+
+``` bash
+python -m examples.message-queue-integrations.solace.quick_start.deploy_core
+```
+
+Deploy the workflow on a terminal.
+
+``` bash
+python -m examples.message-queue-integrations.solace.quick_start.deploy_workflow
+```
+
+Interact with the workflow. The below contains some interaction scenarios:
+
+- The below module generates a task, which sends queries to a workflow sequentially.
+
+``` bash
+python -m examples.message-queue-integrations.solace.quick_start.interaction_simple
+```
+
+- The below command verifies that multiple requests can be handled by a workflow concurrently.
+
+``` bash
+python -m examples.message-queue-integrations.solace.quick_start.interaction_concurrent
+```
+
+- The below command verifies that multiple workflows can run simultaneously.
+
+``` bash
+python -m examples.message-queue-integrations.solace.quick_start.interaction_multi_flows
+```
+
```

Comment:
Can't we use their existing example? funny_joke? 
https://github.com/SolaceDev/llama_deploy/tree/ap/add_solace_event_broker/examples/message-queue-integrations/_app/multi_workflows_app


This is also in our docker compose: https://github.com/SolaceDev/llama_deploy/blob/ap/add_solace_event_broker/examples/message-queue-integrations/solace/docker/docker-compose.yml#L87

Reply:
_A summary of our discussion_
The Funny Joke example has been already covered by the docker example (solace/docker). The above example tests some basic workflows in a different manner.